lambdas/service/app.py

```diff
@@ -897,7 +910,8 @@ def get_integrations():
                     # don't adopt this just yet elsewhere in the program.
                     signature(app.catalog_controller.list_catalogs).return_annotation
                 )
-            }
+            },
+            **common_specs.http_504_response
```

Comment:
Isn't there a less pedestrian way to inject the common spec?

Reply:
The `http_504_response` could potentially be appended to all route method specs in `AzulChaliceApp.route()`, however this would form a circular import

```
ImportError: cannot import name 'AppController' from partially initialized module 'azul.chalice' (most likely due to a circular import) (/Users/daniel/repo/azul3/src/azul/chalice.py)
```

Reply:
> however this would form a circular import

I think you should be able to figure that out. Resolving circular imports is a good Python developer's bread-and-butter.